packages/typescript/test/unit/TypescriptMutator.spec.ts

```diff
@@ -105,8 +105,11 @@ describe('TypescriptMutator', () => {
         const mutants = sut.mutate([
           file1,
         ]);
-        expect(mutants.filter(mutant => mutant.mutatorName === 'SourceFileForTest')).lengthOf(1);
-        expect(mutants.filter(mutant => mutant.mutatorName !== 'SourceFileForTest')).lengthOf(2);
+        expect(mutants.map(mutant => mutant.mutatorName)).to.deep.equal([
+          'SourceFileForTest',
+          'FunctionDeclarationForTest',
+          'FunctionDeclarationForTest'
```

Comment:
I would also expect the location of the mutant to be tested, right? Since that's an important detail. We could also split it into 2 tests, one for a `declare function` and one for a `function`

Reply:
Oh sure, that sounds good. +1